test/features/calendar.feature

```diff
@@ -1,217 +1,208 @@
-#             | startDate   | "2021-01-01T00:00:00.000Z"           |
-#             | endDate     | "2022-01-01T00:00:00.000Z"           |
-#             | createdOn   | "2021-01-01T00:00:00.000Z"           |
-#             | updatedOn   | "2021-01-01T00:00:00.000Z"           |
+    Scenario: get events with authenticated request
+        Given authorisation
+        When make a POST request to "/calendar" with:
+            | name        | "Livestream XY"                       |
+            | description | "descriptive Description"             |
+            | url         | "https://domain.com"                  |
+            | platform    | "YouTube"                             |
+            | author      | {"platform":"discord","uid":"hubber"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"            |
+            | endDate     | "2021-01-01T00:00:00.000Z"            |
+        And the response should contain:
+            | documentId | "TYPE:ID" |
+        When make a GET request to "/calendar/{id}"
+        Then the response status code should be 200
+        And the response should contain:
+            | name        | "Livestream XY"                       |
+            | description | "descriptive Description"             |
+            | url         | "https://domain.com"                  |
+            | platform    | "YouTube"                             |
+            | author      | {"platform":"discord","uid":"hubber"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"            |
+            | endDate     | "2021-01-01T00:00:00.000Z"            |
+            | createdOn   | "TYPE:DATE"                           |
+            | updatedOn   | "TYPE:DATE"                           |
+
+    Scenario: create a event without authorization
+        Given make a POST request to "/calendar" with:
+            | name        | "Livestream XY"                       |
+            | description | "descriptive Description"             |
+            | url         | "https://domain.com"                  |
+            | platform    | "YouTube"                             |
+            | author      | {"platform":"discord","uid":"hubber"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"            |
+            | endDate     | "2021-01-01T00:00:00.000Z"            |
+        Then the response status code should be 401
+        And the response should contain:
+            | statusCode | 401            |
+            | message    | "Unauthorized" |
+
+# Scenario: get sorted ongoing and future events
+#     Given authorisation
+#     And make a POST request to "/calendar" with:
+#         | name        | "Livestream XY"                       |
+#         | description | "descriptive Description"             |
+#         | url         | "https://domain.com"                  |
+#         | platform    | "YouTube"                             |
+#         | author      | {"platform":"discord","uid":"hubber"} |
```

Comment:
still commented out as we would `property` in ``property` where `field` `equals` should contain:

Thats a quite complex step-def which only occurs once